tests/dragonfly/seeder/__init__.py

```diff
@@ -150,6 +150,7 @@ def __init__(
         huge_value_target=5,
         huge_value_size=100000,
         seed=None,
+        huge_value_add_only=False,
```

Comment:
leaving this here for visibility. 

We break `mod` operations for big values that way. We do not therefore create a key with many big values because of that as a side effect. However, this does not change the flows/logic we test so it's fine. 

In the future, we should use 2 tables and one should be used for big values. If a key is a big value, operations like `mod` should only `append` and not `remove` from the working key.